### DISCUSSION TITLE
Multi cell row in table ( a cell that takes the space of two cells )

### DISCUSSION THREAD

**Discussion by ghaith-arabkerly** [pdfme/pdfme#583] [Q&A] [answered]
how to make a cell takes space of two cells in pdfme tables ?
![image](https://github.com/user-attachments/assets/9761a296-3afc-40f5-a7f2-f5a9884478f8)
![image](https://github.com/user-attachments/assets/3d1fbb01-10aa-4f92-8f66-39a0e0a3abee)

**Comment by hand-dot** [comment, marked as answer]
I don't know. Currently we don't have such a feature in table schema
